Dockerfile.dapper

```diff
@@ -3,7 +3,7 @@ FROM golang:1.19-alpine3.16 AS dapper
 ARG ARCH=amd64
 
 RUN apk -U add bash coreutils git gcc musl-dev docker-cli vim less file curl wget ca-certificates
-RUN GOPROXY=direct go install golang.org/x/tools/cmd/goimports@gopls/v0.9.5
+RUN GOPROXY=direct go install golang.org/x/tools/cmd/goimports@gopls/latest
```

Comment:
I would like to highlight this change since it could introduce some instability (moving from a pinned version to latest for gopls)

Reply:
reverting this, CI failed